src/v0/util/index.js

```diff
@@ -1643,7 +1645,25 @@ const checkInvalidRtTfEvents = (inputs, destType) => {
  * @param {Function} singleTfFunc - single event transformation function, we'd recommend this to be an async function(always)
  * @returns
  */
-const simpleProcessRouterDest = async (inputs, destType, singleTfFunc) => {
+const simpleProcessRouterDest = async (
+  inputs,
+  destType,
+  singleTfFunc,
+  reqMetadata
+) => {
+  const getEventReqMetadata = event => {
```

Comment:
looks like this defined in multiple places, is it possible to reuse and avoid duplicate code?

Reply:
Refactored.